### PR TITLE
update v1 version to latest in pipeline and CDN url

### PR DESCRIPTION
**Pull request by lakhveerk**

### CODE REVIEW

**Review by erinha — DISMISSED** [review]
Thanks for making these changes!

**Review by AE-MS — APPROVED** [review]
:shipit:

**Review by jadahiya-MSFT — DISMISSED** [review]

File: azure-pipelines.yml [hunk 1]
```diff
@@ -124,7 +124,7 @@ jobs:
           Arguments: 'build-test-app-V1'
           ProjectDirectory: '$(ClientSdkProjectDirectory)'
 
-      - bash: 'node tools/cli/runAppsWithE2ETests.js --envType=teamsV1 --targetClientSdkCheckpoint=1.11.0 --reportFileName=e2e-tests-report-V1'
+      - bash: 'node tools/cli/runAppsWithE2ETests.js --envType=teamsV1 --targetClientSdkCheckpoint=1.12.1 --reportFileName=e2e-tests-report-V1'
```
Comment (erinha) [inline azure-pipelines.yml:127]: I can see the v1 tests are failing (they haven't finished yet).

You are passing targetClientSdkCheckpoint=1.12.1, but I don't think that checkpoint exists in the code. Do you need to update `ClientSdkCheckpoint.v1` to point be '1.12.1' in cypress/utility/helper.ts?

Comment (jadahiya-MSFT) [reply azure-pipelines.yml:127]: Yup was going to add this comment, thanks for beating me to it. We will need to make sure to update the endpoint for the E2E tests with 1.12.1 as well.

Comment (lakhveerk) [reply azure-pipelines.yml:127]: Thank you for pointing this! I just merged a PR with changes in the host SDK

**Comment by erinha** [comment]
FYI - I think you have to make an updated commit to this PR to have this pipeline pick up the latest host SDK changes.

**Review by AE-MS — APPROVED** [review]

**Review by jadahiya-MSFT — APPROVED** [review]

**Pull request merged**